webpack.config.js

```diff
@@ -42,7 +42,7 @@ module.exports = (env, { mode }) => {
       ]
     },
     resolve: {
-      extensions: ['.tsx', '.ts', '.js']
+      extensions: ['.tsx', '.ts', '.js', '.json']
```

Comment:
why we need .json resolver in project?

Reply:
https://webpack.js.org/configuration/resolve/